src/include/storage/write_ahead_log/log_io.h

```diff
@@ -111,6 +111,11 @@ class BufferedLogWriter {
     return size;
   }
 
+  /**
+   * Empties the buffer.
+   */
+  void EmptyBuffer() { buffer_size_ = 0; }
```

Comment:
This should be called `ResetBuffer` to be more clear, and documentation should describe under what scenarios it should be used.